.travis.yml

```diff
@@ -50,10 +48,13 @@ compiler:
   - clang
   - gcc
 script:
-  - make -j2 COVERAGE=1 CXX="$CXX" HDF5_DIR="$HDF5_DIR" PYTHON_DIR="$PYTHON_DIR"
+  - make -j2 COVERAGE=1 CXX="$CXX" HDF5_DIR="$HDF5_DIR" MPI_NAME=openmpi MPI_LIBS=-lmpi
   - make -j2 test
-  - ./example && ./list example.s5
-  - ./python-example.py && ./list python-example.s5
+  - ./sio-example && ./sio-list example.s5
+  - python setup.py install --user
```

Comment:
Installing software should happen earlier, probably during `install`, not during `script`.